File: JSBASIC/140_count_spaces.js
// 140	140_count_spaces.js	Count the number of spaces in a given sentence using a loop.

let str = "JavaScript is fun to learn";
let count = 0;
for (let i = 0; i < str.length; i++) {
    if (str[i] === " ") count++;
}
console.log("String:", str);
console.log("Number of spaces:", count);

File: JavaBasic/140_count_spaces.java
// 140	140_count_spaces.java	Count the number of spaces in a given sentence using a loop.
public class _140_count_spaces {
    public static void main(String[] args) {
        String str = "Java is fun to learn";
        int count = 0;
        for (char c : str.toCharArray()) if (c == ' ') count++;
        System.out.println("Spaces: " + count);
    }
}
